kpi/utils/usage_calculator.py

```diff
@@ -278,3 +278,78 @@ def _get_cache_hash(self):
             return f'user-{self.user.id}'
         else:
             return f'organization-{self.organization.id}'
+
+
+class BillingAndUsageCalculator:
+    def get_effective_user_id(self, organization: Organization) -> int:
+        return organization.owner.organization_user.user.pk
```

Comment:
This could be simplified with `organization.owner_user_object.pk` and use the cache if it is called more than once.